src/css/themes/bootstrap3/yeti/infragistics.theme.css

```diff
@@ -1149,7 +1149,8 @@ body .ui-tooltip {
 .ui-icon-carat-1-ne {
   background-position: -16px 0;
 }
-.ui-icon-carat-1-e {
+.ui-icon-carat-1-e,
```

Comment:
Why is this necessary? Is it a workaround for something?

Reply:
the class “ui-icon-caret-1-e” in the new version of jQueryUI it’s different from the one “ui-icon-carat-1-e” in the old version. 
Now I know it seems odd but there is actually a different ASCI symbol in the class from the new version

Reply:
OMG. I didn't see the `carAt` vs. `carEt`.

Reply:
lol, you are not the only one :)